### ISSUE TITLE
License?

### ISSUE THREAD

**Issue by wbrockman** [AlgorexHealth/hcc-python#1]
I love the idea of making the CMS HCC scores more readily consumable by non-SAS users.  Do you have any plans to put an open-source license on this work?

**Comment by reverendpaco** [comment]
@wbrockman Yes.  We mean this to be open source.  I just talked with my co-worker and we will probably put this under Apache 2.0 license.  I'll get something up (along with a list of outstanding issues I have on this codebase) within a few days.

**Comment by powellc** [comment]
@reverendpaco any chance you can update the license at some point?

**Comment by Bowiemb** [comment]
@reverendpaco Update on the license by chance? Thanks!
